app/api/helpers/export_helpers.py

```diff
@@ -226,10 +226,13 @@ def export_event_json(event_id, settings):
     )
     uploaded_file = UploadedFile(dir_path, dir_path.rsplit('/', 1)[1])
     storage_url = upload(uploaded_file, storage_path)
-
+    # to extract the raw path
+    if 'http://localhost/' in storage_url:
```

Comment:
This is very brittle

Reply:
If I modify the upload function, would break a lot of other modules. should I pass an optional kwarg in the upload function, such `file_path_only` and pass it in the upload function so that it would check that?

Reply:
The solution would be to never store localhost or domain name in the path and constructing the URL on the go when API request is made

Reply:
@iamareebjamal The problem is that the `upload` function is being used for tickets and order invoices. Just to export the event as a zip, If we don't store the domain in the function, it would break the current implementation of them.